server/player/handler.go

```diff
@@ -22,8 +22,8 @@ type Handler interface {
 	HandleMove(ctx *event.Context, newPos mgl64.Vec3, newYaw, newPitch float64)
 	// HandleTeleport handles the teleportation of a player. ctx.Cancel() may be called to cancel it.
 	HandleTeleport(ctx *event.Context, pos mgl64.Vec3)
-	// HandleChangeWorld handles when the player is added to a new world.
-	HandleChangeWorld(old *world.World, new *world.World)
+	// HandleChangeWorld handles when the player is added to a new world. These values may be nil.
```

Comment:
Pretty sure only the new world could be nil here.

Reply:
What if a world gets deleted?

Reply:
Unless you mean old, that makes sense

Reply:
No, I mean new, do you mean deleted as in closed?

Reply:
Wait, I mean old, I just confused the shit out of myself